src/sg/gov/moh/healthcert/1.0/schema.json

```diff
@@ -313,18 +343,22 @@
               "minItems": 1
             },
             "address": {
+              "description": "Visiting or postal addresses for the contact.",
               "type": "object",
               "required": ["type", "use", "text"],
               "properties": {
                 "type": {
-                  "type": "string",
+                  "description": "Distinguishes between physical addresses (those you can visit) and mailing addresses (e.g. PO Boxes and care-of addresses). Most addresses are both.",
+                  "enum": ["postal", "physical", "both"],
                   "examples": ["work"]
                 },
                 "use": {
-                  "type": "string",
+                  "description": "The purpose of this address.",
+                  "enum": ["home", "work", "temp", "old", "billing"],
                   "examples": ["physical"]
```

Comment:
This example value also should be updated, shouldn't it. Or eliminated.

Reply:
updated